### DISCUSSION TITLE
How can I create a cuda enabled project which using NVCC compiler?

### DISCUSSION THREAD

**Discussion by FantasyVR** [premake/premake-core#2144] [Q&A]
I want to compile `.cu` cuda files with the NVCC compiler. I don't know how to configure the compiler using premake.

**Comment by nickclark2016** [comment]
We don't have CUDA support out of the box. I recommend checking out some plugins that add the functionality. I have not used this personally, but here's a plugin I found from a quick google search: https://github.com/theComputeKid/premake5-cuda
